pkg/api/v1/types.go

```diff
@@ -2251,6 +2259,9 @@ type Event struct {
 
 	// The number of times this event has occurred.
 	Count int `json:"count,omitempty"`
+
+	// Type of this event (Normal, Warning), new types could be added in the future
+	Type string `json:"type,omitempty"`
```

Comment:
It's intentionally ambiguous. I don't want this to evolve in the direct of log severity levels, which are very hard to use in my experience. It's just a coarse bucketing of Reasons along some useful dimensions, TBD. Like Reason, it shouldn't be considered to be an enum.

The Real Solution to filtering is something like separately defined alerting rules.